.requirements

```diff
@@ -7,3 +7,4 @@ RESTY_LUAROCKS_VERSION=3.2.1
 RESTY_OPENSSL_VERSION=1.1.1d
 RESTY_PCRE_VERSION=8.43
 LIBYAML_VERSION=0.2.2
+KONG_GO_PLUGINSERVER_VERSION=master
```

Comment:
I think we should tag a release number there and pin it here before merging.

Reply:
Done!